File: frontapp/src/app/model/examination.ts
import { Pharmacy } from "./pharmacy";
import { User } from "./user";

export class Examination {
    id: number;
    price: number;
    dateAndTime: Date;
    dermatologist: User;
    patient: User;
    pharmacy: Pharmacy;
    executed: boolean;
    didntShow: boolean;
    report: string;
}

File: apoteka/src/main/java/com/back/apoteka/model/Examination.java
package com.back.apoteka.model;



import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.Data;

import java.sql.Timestamp;


@Data
@Entity
@Table(name = "examination")
public class Examination {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	
	private Timestamp dateAndTime;
	@ManyToOne(fetch = FetchType.EAGER)
	private User dermatologist;
	
    @ManyToOne(fetch = FetchType.EAGER)
	private Pharmacy pharmacy;
    
    @ManyToOne(fetch = FetchType.EAGER)
    private User patient;
	
    private boolean executed;
    private boolean didntShow;
    private String report;  
	private double price;

}
